tests/test_complex_questions.py

```diff
@@ -142,19 +150,23 @@ def test_api(tmp_path, template_path):
 def test_cli_interactive(tmp_path, spawn, template_path):
     """Test copier correctly processes advanced questions and answers through CLI."""
 
-    def check_invalid(tui, name, format, invalid_value, help=None):
+    def check_invalid(tui, name, format, invalid_value, help=None, err="Invalid input"):
         """Check that invalid input is reported correctly"""
         expect_prompt(tui, name, format, help)
         tui.sendline(invalid_value)
         tui.expect_exact(invalid_value)
-        tui.expect_exact("Invalid input")
+        tui.expect_exact(err)
 
     tui = spawn(COPIER_PATH + ("copy", template_path, str(tmp_path)), timeout=10)
     expect_prompt(tui, "love_me", "bool", help="I need to know it. Do you love me?")
     tui.send("y")
     expect_prompt(tui, "your_name", "str", help="Please tell me your name.")
     tui.sendline("Guybrush Threpwood")
     check_invalid(tui, "your_age", "int", "wrong your_age", help="How old are you?")
+    # tui.send((Keyboard.Alt + Keyboard.Backspace) * 2)
```

Comment:
can't for the life of me debug the timeout here.  also tried creating a new field (not `your_age`).  Help from anyone more experienced with pexpect would be appreciated :)

Reply:
Hehe don't worry! Indeed pexpect is hard to deal with.

I'll tell you how. The trick is in the `spawn` fixture: https://github.com/copier-org/copier/blob/241b28fb7ab179400c48c608bd42c41e4d1c5185/tests/conftest.py#L28-L34

Put simply, it will remove the timeout if it detects you're running a debugging session.

If you don't know how to set up properly the debugger, try out [hacking with Gitpod](https://github.com/copier-org/copier/blob/master/CONTRIBUTING.md#dev-environment-setup), which is preconfigured (well, I don't remember if you'll need to install the python extension, maybe). Then you can just trigger the test in debug mode with 1 click, and the timeout will be magically slurped.

But if you can't still get it, then don't worry, just do the functionality and I'll fix the test.